### PR TITLE
chore: remove charts directory

### LINKED ISSUE #987: Remove unused charts and dockers directory and files
### Requested Feature

```markdown
Remove unused charts and dockers directory and files since they have their own dedicated repos now.
```

### PR DESCRIPTION
**Pull request by JeffreyDallas**
## Description

This pull request changes the following:

- Remove charts directory
- Remove .github/workflow/support directory since the chart test is being conducted in `solo-charts` repo 
- Update workflow and unit tests

### Related Issues

- Closes #987

### CODE REVIEW

**Comment by JeffreyDallas** [comment]
@jeromy-cannon since Helm test workflow is removed,
`FS Helm Chart Tests` did not kick off
need your manually override to merge it

File: .github/workflows/zxc-helm-chart-tests.yaml
Comment (jeromy-cannon) [inline .github/workflows/zxc-helm-chart-tests.yaml:1]: shouldn't we delete this file?

Comment (JeffreyDallas) [reply .github/workflows/zxc-helm-chart-tests.yaml:1]: updated

File: fullstack-gradle-plugin/src/test/kotlin/com/hedera/fullstack/gradle/plugin/HelmDependencyUpdateTaskTest.kt
Comment (jeromy-cannon) [inline fullstack-gradle-plugin/src/test/kotlin/com/hedera/fullstack/gradle/plugin/HelmDependencyUpdateTaskTest.kt:1]: I think this is good to leave in for now.

File: fullstack-examples/build.gradle.kts [hunk 1]
```diff
@@ -58,10 +50,6 @@ tasks.register<HelmReleaseExistsTask>("helmNginxExists") {
     release.set("nginx-release")
 }
 
-tasks.register<HelmDependencyUpdateTask>("helmDependencyUpdate") {
```
Comment (jeromy-cannon) [inline fullstack-examples/build.gradle.kts:61]: I think this is good to leave in for now.  I think you can just run it on one of the other charts we already installed.

Comment (JeffreyDallas) [reply fullstack-examples/build.gradle.kts:61]: what are the other charts installed ?
since the whole charts directory is gone

Comment (jeromy-cannon) [reply fullstack-examples/build.gradle.kts:61]: `nginx-release`, but it doesn't really matter.  it is just an example of how to use the gradle plugin task.

File: fullstack-gradle-plugin/src/test/kotlin/com/hedera/fullstack/gradle/plugin/HelmDependencyUpdateTaskTest.kt
Comment (JeffreyDallas) [reply fullstack-gradle-plugin/src/test/kotlin/com/hedera/fullstack/gradle/plugin/HelmDependencyUpdateTaskTest.kt:1]: updated

File: fullstack-examples/build.gradle.kts (hunk 1 above)
Comment (jeromy-cannon) [reply fullstack-examples/build.gradle.kts:61]: what about if you use the solo-charts?

Comment (JeffreyDallas) [reply fullstack-examples/build.gradle.kts:61]: Changed to using solo-charts

Comment (jeromy-cannon) [reply fullstack-examples/build.gradle.kts:61]: are you able to get helm dependency update task to work?

Comment (JeffreyDallas) [reply fullstack-examples/build.gradle.kts:61]: Could not get it work.
It seems only worked with a local directory chart

File: fullstack-gradle-plugin/src/test/kotlin/com/hedera/fullstack/gradle/plugin/HelmDependencyUpdateTaskTest.kt [hunk 2]
```diff
@@ -39,21 +39,7 @@ class HelmDependencyUpdateTaskTest {
             project = ProjectBuilder.builder().build()
         }
     }
-
-    @Test
-    @DisplayName("Test Helm Dependency Update Task")
```
Comment (jeromy-cannon) [inline fullstack-gradle-plugin/src/test/kotlin/com/hedera/fullstack/gradle/plugin/HelmDependencyUpdateTaskTest.kt:44]: don't delete this.

File: fullstack-examples/build.gradle.kts (hunk 1 above)
Comment (jeromy-cannon) [reply fullstack-examples/build.gradle.kts:61]: okay, we can drop the test coverage for now, but keep @DisplayName("Test Helm Dependency Update Task")

**Review by jeromy-cannon — APPROVED** [review]

**Pull request merged**